packages/cli/src/commands/generate/generator/generator.js

```diff
@@ -0,0 +1,97 @@
+import path from 'path'
+
+import fse from 'fs-extra'
+import Listr from 'listr'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from '../../../lib'
+import c from '../../../lib/colors'
+
+export const command = 'generator <name>'
+export const description =
+  'Copies generator templates locally for customization'
+
+const SIDE_MAP = {
```

Comment:
Crazy idea 2: what if our built generators were also organised into folders? You could then look up which side it's in

Reply:
Hmm, explain? Do you mean the templates that are in redwoodjs/cli are organized into web/api directories instead of inside each individual generator's directory? Right now they're like:

![image](https://user-images.githubusercontent.com/300/133529603-f06ad427-9461-4591-9fc1-ed99edc23412.png)

Reply:
Yeah - so be consistent with how we do custom templates. 

Put all of the web stuff into generate/web/ and api stuff into generate/api/ - basically mirror the folder structure we would be creating in the users project.

Might be a lot of work, and not sure what yargs would do - but just an idea! Also would mean no changes when new generators get added 

Alternatively - we could have custom templates at the root of the project instead of on individual sides, which would remove the need for this

Reply:
Oh mannnnnn seems like a bunch of changessssss...also the scaffold generator kind of does both (triggers SDL and service generators). 

How about an issue that proposes a future improvement to this functionality. ;)

Reply:
Yeah sensible - I wonder if there's a simpler way to achieve this without using folders. 🤔